seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -231,16 +271,36 @@ angular.module('BE.seed.controller.data_quality_admin', [])
         spinner_utility.show();
         data_quality_service.save_data_quality_rules($scope.org.org_id, rules).then(function (rules) {
           loadRules(rules);
-          $scope.rules_updated = true;
           modified_service.resetModified();
         }).then(function (data) {
           $scope.$emit('app_success', data);
         }).catch(function (data) {
           $scope.$emit('app_error', data);
         }).finally(function () {
+          $scope.rules_updated = true;
           spinner_utility.hide();
         });
       };
+      $scope.change_condition = function (rule) {
+        $scope.rules_updated = false;
+        $scope.defaults_restored = false;
+        $scope.rules_reset = false;
+        if (!rule.condition) rule.condition = null;
```

Comment:
Line 288 is since we don't have a 'null' condition as an option.
Ln 285 - 287 is to hide the checked mark if condition is being changed:
![Screen Shot 2020-05-07 at 4 41 42 PM](https://user-images.githubusercontent.com/49968203/81351647-13c44700-9082-11ea-83ad-fd4221ebb508.png)